### PR TITLE
Adding CI support for examples

### PR DESCRIPTION
**Pull request by bfosberry**

### CODE REVIEW

**Comment by mlocher** [comment]
I updated the project at https://codeship.com/projects/103513 to use @bfosberry's GitHub authentication, so we should see PR status from now on.

**Comment by mlocher** [comment]
Is there any way for us to use the services and steps files from the examples directly and not having to duplicate all the information into the top level `codeship-services.yaml` and `codeship-steps.yaml` files? I fear that they will get out of sync otherwise.

Should we add the top level `codeship.aes` to `.gitignore`?

And what about missing examples (e.g. 06, do we have plans to add those as well)

Other than that, awesome work and :shipit:

**Comment by bfosberry** [comment]
There is currently no way to use child steps and services files, we should add the key and tmp, and 6 looks like it runs for me, what do you see?

File: codeship-steps.yml [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+- service: 17demo
+  name: 17
+  command: ls -l
+- name: 18
+  service: 18test
+  command: ab -n 100 -c 15 http://lb/
+- name: 19
+  service: 19demo
+  type: push
+  name: 20
```
Comment (mlocher) [inline codeship-steps.yml]: missing a `-` here

**Comment by mlocher** [comment]
You're right, example 6 is running fine, I didn't see that it was using the containers from example 5.

Examples 12 and 13 don't exist (which is what I was going to ask next, as I mostly looked at the services and steps file).

All the rest is :+1:

**Comment by bfosberry** [comment]
Not sure why those examples dont exist :P

File: codeship-steps.yml (hunk 1 above)
Comment (bfosberry) [inline codeship-steps.yml]: huh, that looks like a mistake

**Comment by mlocher** [comment]
:shipit:

**Pull request merged**